upi/vsphere/install-config.yaml.example

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+baseDomain:
+metadata:
+  name:
+networking:
+  machineCIDR: ""
+platform:
+  vsphere:
+    vCenter:
+    username:
+    password:
+    datacenter:
+    defaultDatastore:
+pullSecret:
+sshKey:
```

Comment:
nit: did we want to fill these in with placeholders/examples like [these examples][1]?

[1]: https://github.com/openshift/installer/blob/8e75675f373d7ca46ec7fa4b4156ce75287735d1/docs/user/customization.md#additional-trust-bundle